metadata-ingestion/src/datahub/ingestion/source/usage/usage_common.py

```diff
@@ -251,3 +258,54 @@ def generate_workunits(
                     resource_urn_builder=resource_urn_builder,
                     user_urn_builder=user_urn_builder,
                 )
+
+
+def convert_usage_aggregation_class(
```

Comment:
Fine leaving this in here -- we can remove this when we remove `UsageAggregationClass` entirely

Reply:
sounds good - let's add a TODO / deprecation marker on this though